sdk/src/azure/core/az_context.c

```diff
@@ -33,6 +36,9 @@ AZ_NODISCARD int64_t az_context_get_expiration(az_context const* context)
 AZ_NODISCARD az_result
 az_context_get_value(az_context const* context, void const* key, void const** out_value)
 {
+  _az_PRECONDITION_NOT_NULL(context);
+  _az_PRECONDITION_NOT_NULL(key);
```

Comment:
```suggestion
  _az_PRECONDITION_NOT_NULL(key);
  _az_PRECONDITION_NOT_NULL(*out_value);
```

Reply:
Added along with a check that out_value itself is not null